chain/x/oracle/module.go

```diff
@@ -49,199 +56,149 @@ func (AppModuleBasic) ValidateGenesis(cdc codec.JSONMarshaler, bz json.RawMessag
 		AppModuleBasic: AppModuleBasic{},
 		keeper:         k,
 	}
 }
 
-func (AppModule) Name() string {
-	return ModuleName
-}
-
+// RegisterInvariants implements the AppModule interface.
 func (am AppModule) RegisterInvariants(ir sdk.InvariantRegistry) {}
 
-func (am AppModule) Route() string {
-	return RouterKey
-}
+// Route implements the AppModule interface.
+func (am AppModule) Route() string { return RouterKey }
 
-func (am AppModule) NewHandler() sdk.Handler {
-	return NewHandler(am.keeper)
-}
-func (am AppModule) QuerierRoute() string {
-	return ModuleName
-}
+// NewHandler implements the AppModule interface.
+func (am AppModule) NewHandler() sdk.Handler { return NewHandler(am.keeper) }
 
-func (am AppModule) NewQuerierHandler() sdk.Querier {
-	return NewQuerier(am.keeper)
-}
+// QuerierRoute implements the AppModule interface.
+func (am AppModule) QuerierRoute() string { return ModuleName }
 
+// NewQuerierHandler implements the AppModule interface.
+func (am AppModule) NewQuerierHandler() sdk.Querier { return NewQuerier(am.keeper) }
+
+// BeginBlock implements the AppModule interface.
 func (am AppModule) BeginBlock(_ sdk.Context, _ abci.RequestBeginBlock) {}
 
+// EndBlock implements the AppModule interface.
 func (am AppModule) EndBlock(ctx sdk.Context, _ abci.RequestEndBlock) []abci.ValidatorUpdate {
 	handleEndBlock(ctx, am.keeper)
 	return []abci.ValidatorUpdate{}
 }
 
+// InitGenesis performs genesis initialization for the oracle module.
 func (am AppModule) InitGenesis(ctx sdk.Context, cdc codec.JSONMarshaler, data json.RawMessage) []abci.ValidatorUpdate {
 	var genesisState GenesisState
 	cdc.MustUnmarshalJSON(data, &genesisState)
 	return InitGenesis(ctx, am.keeper, genesisState)
 }
 
+// ExportGenesis returns the current state as genesis raw bytes.
 func (am AppModule) ExportGenesis(ctx sdk.Context, cdc codec.JSONMarshaler) json.RawMessage {
 	gs := ExportGenesis(ctx, am.keeper)
 	return cdc.MustMarshalJSON(gs)
 }
 
-// Implement IBCModule callbacks
+// OnChanOpenInit implements ics-26 IBCModule interface.
 func (am AppModule) OnChanOpenInit(
-	ctx sdk.Context,
-	order channelexported.Order,
-	connectionHops []string,
-	portID string,
-	channelID string,
-	chanCap *capability.Capability,
-	counterparty channeltypes.Counterparty,
-	version string,
+	ctx sdk.Context, order channelexported.Order, connectionHops []string,
+	portID string, channelID string, chanCap *capability.Capability,
+	counterparty channeltypes.Counterparty, version string,
 ) error {
-	// // TODO: Enforce ordering, currently relayers use ORDERED channels
-	// if version != types.Version {
-	// 	return sdkerrors.Wrapf(porttypes.ErrInvalidPort, "invalid version: %s, expected %s", version, "ics20-1")
-	// }
-
-	// Claim channel capability passed back by IBC module
-	if err := am.keeper.ScopedKeeper.ClaimCapability(ctx, chanCap, ibctypes.ChannelCapabilityPath(portID, channelID)); err != nil {
+	// Claim channel capability passed back by IBC module.
```

Comment:
Remove version?

Reply:
Yeah I don't think it's needed for us. Let's just skip it.